frontend/public/components/overview/pods-overview.tsx

```diff
@@ -0,0 +1,148 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { ListGroup } from 'patternfly-react';
+
+import { ResourceLink, resourcePath, SidebarSectionHeading, StatusIconAndText } from '../utils';
+import { podPhase, PodKind, K8sResourceKind } from '../../module/k8s';
+
+const MAX_PODS: number = 3;
+const MAX_ERROR_PODS: number = 10;
+
+const podUpdateTime = (pod: PodKind ) => {
+  const allStatuses = [
+    ..._.get(pod, 'status.containerStatuses', []),
+    ..._.get(pod, 'status.initContainerStatuses', []),
+  ];
+  const updateTimes = _.reduce(allStatuses, (times, nextStatus) => {
+    if (nextStatus.state.running) {
+      return [...times, _.get(nextStatus, 'state.running.startedAt')];
+    }
+    if (nextStatus.state.terminated) {
+      return [...times, _.get(nextStatus, 'state.terminated.finishedAt')];
+    }
+    if (nextStatus.lastState.running) {
+      return [...times, _.get(nextStatus, 'lastState.running.startedAt')];
+    }
+    if (nextStatus.lastState.terminated) {
+      return [...times, _.get(nextStatus, 'lastState.terminated.finishedAt')];
+    }
+    return [...times, _.get(nextStatus, _.get(pod, 'startTime'))];
+  }, []);
+
+  return _.head(_.reverse(updateTimes.sort()));
+};
+
+const errorPhases = [
+  'ContainerCannotRun',
+  'CrashLoopBackOff',
+  'Critical',
+  'Error',
+  'Failed',
+  'InstallCheckFailed',
+  'Cancelled',
+  'Expired',
+  'Not Ready',
```

Comment:
This isn't an error phase. Essentially it means the pod is still starting.

If this is just a list of pods we want to sort to the top, that's OK. I'd suggest a different variable name, maybe `interestingPhases`? (I'm not great with names :)